src/blueprints/tv2/timeline-object-factories/tv2-tri-caster-video-mixer-timeline-object-factory.ts

```diff
@@ -355,7 +355,7 @@ export class Tv2TriCasterVideoMixerTimelineObjectFactory implements Tv2VideoMixe
   }
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  public createTBarTransitionEffectTimelineObject(_sourceInput: number, _tBarPosition: number): Tv2BlueprintTimelineObject {
+  public createTBarTransitionEffectTimelineObject(_fromSourceInput: number, _toSourceInput: number, _tBarPosition: number): Tv2BlueprintTimelineObject {
     throw new Error('Not implemented!')
   }
```

Comment:
Suggestion, so we can get a little usefulness out of the "not implemented for now 😄 Means we can get rid of the eslint-disabling too

`public createTBarTransitionEffectTimelineObject(_fromSourceInput: number, _toSourceInput: number, _tBarPosition: number): Tv2BlueprintTimelineObject {
    throw new Error(`Not implemented, but got ${_fromSourceInput}, ${_toSourceInput} and ${_tBarPosition}`)
  }`

Reply:
The eslint rule for allowing unused variables that start with `_` should be in place, so the eslint-disable-next-line can be removed nonetheless :)

Reply:
The disable line has been removed